ci-operator/step-registry/ipi/alibabacloud/post/OWNERS

```diff
@@ -0,0 +1 @@
+../OWNERS
```

Comment:
Not sure, just me fumbling around. @rvanderp3 any idea on this one?

Reply:
The ci-operator tooling requires an explicit `OWNERS` in each `ci-operator/step-registry` directory, since at least #9739 when I started using symlinks to reduce some of the maintenance tedium around that.  Possibly worth a ci-tools RFE about removing that requirement?